admin/class-smaily-for-cf7-admin.php

```diff
@@ -278,36 +277,35 @@ public function verify_credentials_callback() {
 			wp_send_json( $response );
 		}
 
-		$subdomain     = isset( $_POST['subdomain'] ) ? trim( $_POST['subdomain'] ) : null;
-		$username      = isset( $_POST['username'] ) ? trim( $_POST['username'] ) : null;
-		$password      = isset( $_POST['password'] ) ? trim( $_POST['password'] ) : null;
+		$subdomain     = isset( $_POST['subdomain'] ) ? sanitize_text_field( wp_unslash( $_POST['subdomain'] ) ) : null;
+		$username      = isset( $_POST['username'] ) ? sanitize_text_field( wp_unslash( $_POST['username'] ) ) : null;
+		$password      = isset( $_POST['password'] ) ? sanitize_text_field( wp_unslash( $_POST['password'] ) ) : null;
 		$autoresponder = isset( $_POST['autoresponder'] ) ? (int) $_POST['autoresponder'] : 0;
 
-		$sanitized              = $this->sanitize_credentials( $subdomain, $username, $password );
-		$sanitized['subdomain'] = $this->normalize_subdomain( $sanitized['subdomain'] );
+		$subdomain = $this->normalize_subdomain( $subdomain );
 
 		if (
-			empty( $sanitized['subdomain'] )
-			|| empty( $sanitized['username'] )
-			|| empty( $sanitized['password'] )
+			empty( $subdomain )
```

Comment:
Same here. Move if-condition clauses to single line. It makes it a bit easier on the eyes.